[IMP] l10n_es: add CoA for Cooperativas

dylankiss opened the pull request:
In Spain, "Cooperativas" have their own variant of the Spanish chart of accounts. In order to support these businesses, we add two extra CoA templates: Cooperatives - SMEs and Cooperatives - Complete.

[task-3803050](https://www.odoo.com/web#id=3803050&cids=1&model=project.task&view_type=form)

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/159357).

pedrobaeza commented:
It would be `Cooperativas` instead of `Cooperativos`, hehe.

@acysos can you check that this is OK?

dylankiss commented:
> It would be `Cooperativas` instead of `Cooperativos`, hehe.

@pedrobaeza Thanks for the correction ;) Fixed

jco-odoo commented on addons/l10n_es/data/template/account.account-es_pymes.csv:
```diff
@@ -1,37 +1,45 @@
-"account_pymes_1044","Uncalled non-monetary contributions, capital pending registration","Socios por aportaciones no dinerarias pendientes, capital pendiente de inscripción","1044","liability_non_current"
-"account_pymes_120","Retained earnings","Remanente","120","liability_non_current"
-"account_pymes_121","Prior period's losses","Resultados negativos de ejercicios anteriores","121","liability_non_current"
-"account_pymes_129","Profit/loss for the period","Resultado del ejercicio","129","equity_unaffected"
```
We recently changed this to equity_unaffected, so it would be nice to keep that change.

dylankiss replied:
What do you mean? It's still `equity_unaffected`. I just added the `reconcile` column here.

jco-odoo replied:
indeed, looking at the wrong line

jco-odoo commented:
@robodoo r+

dylankiss commented:
@jco-odoo There were conflicts indeed of a recently merged fix where some accounts were changed to `equity`. I just rebased and fixed the conflicts. Can your re-r+?

jco-odoo commented:
@robodoo r+
